ani-cli

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+# version umber
+VERSION="1.0"
```

Comment:
Actually please set 1.0.0 since three numbers seem to be best practice here on github

Reply:
actually its such a small change I can do that myself